<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackSettingsPage</class>
 <widget class="QWidget" name="PlaybackSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playback</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QCheckBox" name="current_glow">
     <property name="text">
      <string>Show a glowing animation on the current track</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fading_groupBox">
     <property name="title">
      <string>Fading</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="fading_out">
        <property name="text">
         <string>Fade out when stopping a track</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fading_cross">
        <property name="text">
         <string>Cross-fade when changing tracks manually</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fading_auto">
        <property name="text">
         <string>Cross-fade when changing tracks automatically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fading_samealbum">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Except between tracks on the same album or in the same CUE sheet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fading_options" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Fading duration</string>
           </property>
           <property name="indent">
            <number>22</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fading_duration">
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>2000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>257</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fadeout_pause">
        <property name="text">
         <string>Fade out on pause / fade in on resume</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Fading duration</string>
          </property>
          <property name="indent">
           <number>22</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fading_pause_duration">
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>50</number>
          </property>
          <property name="value">
           <number>250</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="power_management_groupBox">
     <property name="title">
      <string>System Power Management Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="inhibit_suspend_while_playing">
        <property name="toolTip">
         <string>If enabled, Clementine will prevent the system from automatically suspending if it is playing a track</string>
        </property>
        <property name="text">
         <string>Inhibit automatic suspend if playing</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="replaygain_group">
     <property name="title">
      <string>Replay Gain</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="replaygain">
        <property name="text">
         <string>Use Replay Gain metadata if it is available</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Replay Gain mode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="replaygain_mode">
           <item>
            <property name="text">
             <string>Radio (equal loudness for all tracks)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Album (ideal loudness for all tracks)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Pre-amp</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="replaygain_preamp_label"/>
           </item>
           <item>
            <widget class="StickySlider" name="replaygain_preamp">
             <property name="maximum">
              <number>300</number>
             </property>
             <property name="value">
              <number>150</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sticky_center" stdset="0">
              <number>150</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="replaygain_compression">
           <property name="text">
            <string>Apply compression to prevent clipping</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gst_group">
     <property name="title">
      <string>Audio output</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Output device</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="gst_output"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="buffer_duration_label">
        <property name="text">
         <string>Buffer duration</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="buffer_duration">
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="maximum">
         <number>60000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="buffer_min_fill_label">
        <property name="text">
         <string>Minimum buffer fill</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="buffer_min_fill_value_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="buffer_min_fill">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="value">
           <number>33</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="mono_playback">
        <property name="toolTip">
         <string>Changing mono playback preference will be effective for the next playing songs</string>
        </property>
        <property name="text">
         <string>Mono playback</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sample_rate_label">
        <property name="text">
         <string>Sample rate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="sample_rate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Changes will take place when the next song starts playing</string>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Auto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>44,100Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>48,000Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>96,000Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>192,000Hz</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>114</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StickySlider</class>
   <extends>QSlider</extends>
   <header>widgets/stickyslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>replaygain</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fading_auto</sender>
   <signal>toggled(bool)</signal>
   <receiver>fading_samealbum</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
